api/drf_views.py

```diff
@@ -809,8 +817,8 @@ def save_meta(self, fieldreport, meta, is_update=False):
 
 
 class CreateFieldReport(CreateAPIView, GenericFieldReportView):
-    authentication_classes = (TokenAuthentication,)
-    permission_classes = (IsAuthenticated,)
+    #authentication_classes = (TokenAuthentication,)
```

Comment:
Are you sure that authentication is not required to create a field report?
(I have a fear that robots will create field reports this way.)